lib/spree/chimpy/interface/orders.rb

```diff
@@ -1,102 +1,257 @@
 module Spree::Chimpy
   module Interface
     class Orders
+      include Rails.application.routes.url_helpers
       delegate :log, to: Spree::Chimpy
 
       def initialize
```

Comment:
If there isn't anything that is run during the initialize method call, we can probably just remove it during this refactoring. What yah think?